llvm/test/CodeGen/PowerPC/copysignl.ll

```diff
@@ -91,8 +92,9 @@ define ppc_fp128 @foo_ld(double %a, double %b) #0 {
 ; CHECK-VSX-NEXT:    fmr 0, 1
 ; CHECK-VSX-NEXT:    xscpsgndp 1, 2, 1
 ; CHECK-VSX-NEXT:    xxlxor 2, 2, 2
-; CHECK-VSX-NEXT:    xscmpudp 0, 0, 1
-; CHECK-VSX-NEXT:    beqlr 0
+; CHECK-VSX-NEXT:    fcmpu 0, 0, 1
+; CHECK-VSX-NEXT:    cror 20, 2, 3
+; CHECK-VSX-NEXT:    bclr 12, 20, 0
```

Comment:
@workingjubilee please can you rebase after 32aa782ea297b3e0ec090cf8fc0055d00c99d24b to more clearly see the diff?

Reply:
I already did that in my prior commit...

Reply:
@RKSimon If I rebase on main, which I have to now, that will just dump 1000 files of diff on it in the GH view. It will obliterate any ability to see anything. If you wanted me to insert a commit at the start, before any code changes, I could have simply done that.

Reply:
I know how to shove the rebase several times to make it so that GitHub renders a minimal diff eventually anyways, but **please never do this to anyone again.**

Reply:
If you have write access, you could just shove my branch and get the commit where you want it anyways.

Reply:
> If I rebase on main, which I have to now, that will just dump 1000 files of diff on it in the GH view.

No it won't

Reply:
If you look at the Files view, yes, but it removes the ability to compare between the rebase versions.

Reply:
Yes, but it doesn't really matter. I care about that < 2% of the time and have never looked at the GitHub version comparison

Reply:
I suppose.

LLVM's a verrry long rebuild on my machine though, I'm not using a 100-core monster.